classifiers/python_functions/vader_sentiment/__init__.py

```diff
@@ -0,0 +1,32 @@
+from pydantic import BaseModel
+from vaderSentiment.vaderSentiment import SentimentIntensityAnalyzer
+
+INPUT_EXAMPLE = {
+    "text": "World peace announced by the United Nations.",
+    "mode": "classification" # Choose "scores" to only get the sentiment scores as floats
+}
+
+class VaderSentimentModel(BaseModel):
+    text: str
+    mode: str
+
+    class Config:
+        schema_extra = {"example": INPUT_EXAMPLE}
+
+def vader_sentiment(req):
+    '''Get the sentiment of a text using the VADER algorithm.'''
+    analyzer = SentimentIntensityAnalyzer()
+    text = req.text
+
+    vs = analyzer.polarity_scores(text)
+    if req.mode == "classification":
+        if vs["compound"] >= 0.05:
+            return {"sentiment": "positive"}
+        elif vs["compound"] > -0.05: 
```

Comment:
I might be wrong, but in this elif --> `vs["compound"] > -0.05 and vs["compound"] < 0.05` makes it complete, doesn't it? Otherwise two if statements being right at the same time would lead to problems :)

Reply:
Nah I think it should be fine. Took this from the creat